lib/pythontarget.c

```diff
@@ -679,6 +725,25 @@ action_capsule_init(generic_action_capsule_struct *self, PyObject *args, PyObjec
     return 0;
 }
 
+
+///////////////// Functions used in Tag creation, initialization and deletion /////////////
+/**
+ * Initialize the Tag object with the given values for "time" and "microstep", 
+ * both of which are required.
+ * @param self A py_tag_t object.
+ * @param args The arguments are:
+ *      @param time A logical time.
+ *      @param microstep A microstep within the logical time "time".
+ */
+static int Tag_init(py_tag_t *self, PyObject *args, PyObject *kwds)
+{
```

Comment:
I think by convention, the ` {` should be on the same line as the function (note the space before `{`). This is not always followed in the Python support files unfortunately, but I think we should try to follow it as best as we could.